docs/support/troubleshooting.md

```diff
@@ -14,11 +14,27 @@ change something.
 
 Please see [this](../scanning_issues/) page on scanning issues.
 
+
+### I can't enter leading zeros for my PIN
```

Comment:
This is instructions for the Home Assistant component and will have to be added to the Home Assistant documentation later. Here we should only care for `pyatv` stuff.